src/smt.rs

```diff
@@ -447,13 +442,39 @@ impl rocksdb::WriteBatchIterator for UpdateData {
     }
 }
 
+impl QueryProof {
```

Comment:
The same applies here regarding inline functions.

Reply:
use `inline` tag that explained in above comment.